doc/source/cluster/key-concepts.rst

```diff
@@ -60,19 +60,31 @@ sit idle, the autoscaler will remove worker nodes from the cluster.
 It is important to understand that the autoscaler only reacts to task and actor resource requests, and not application metrics or physical resource utilization.
 To learn more about autoscaling, refer to the user guides for Ray clusters on :ref:`VMs <cloud-vm-index>` and :ref:`Kubernetes <kuberay-index>`.
 
+
+.. _ray_driver:
+
+Ray Driver
+----------
+
+A Ray driver is the process that calls the Ray tasks and actors in the user application.
+A driver is started by running `ray.init()`. The script that calls `ray.init()` may also be referred to as the "driver script".
+
+
 .. _cluster-clients-and-jobs:
 
-Ray Jobs
---------
+Ray Job
+-------
+
+A Ray Job is a user application that includes a Ray driver. There are two ways of running Ray Jobs on Ray clusters:
```

Comment:
This definition is not very precise? It makes it sound like there are user applications that do not include Ray drivers.

Suggested (adapted from whitepaper):
A Ray job is a single application: it is the collection of Ray tasks, objects, and actors that originate from the same entrypoint. The worker that runs the Python entrypoint is known as the *driver* of the job.

Reply:
Maybe we can call it script?

Reply:
script is clearer to me.

Reply:
Thanks, I've added the new definition